### ISSUE TITLE
fileshandling/file.go:31:14: invalid operation: kind != "gopkg.in/h2non/filetype.v1/types".Unknown (mismatched types "github.com/h2non/filetype/types".Type and "gopkg.in/h2non/filetype.v1/types".Type)

### ISSUE THREAD

**Issue by fujii** [nmrshll/gphotos-uploader-cli#38]
I saw the following build error.

~~~
$ go get -u github.com/nmrshll/gphotos-uploader-cli/cmd/gphotos-uploader-cli
# github.com/nmrshll/gphotos-uploader-cli/fileshandling
go/src/github.com/nmrshll/gphotos-uploader-cli/fileshandling/file.go:31:14: invalid operation: kind != "gopkg.in/h2non/filetype.v1/types".Unknown (mismatched types "github.com/h2non/filetype/types".Type and "gopkg.in/h2non/filetype.v1/types".Type)
~~~

So, I installed v0.1.11 instead.

**Comment by troycurtisjr** [comment]
I had the same error on Fedora 29 using `go version go1.11.4 linux/amd64`.

**Comment by nmrshll** [comment]
Closed in #39
